dlc_developer_config.toml

```diff
@@ -0,0 +1,28 @@
+[dev]
+# Set to "huggingface", for example, if you are a huggingface developer
+partner_developer = ""
```

Comment:
Should we add the expected values for `partner_developer` in the comments?

Reply:
added